applications/nrf_desktop/doc/dvfs.rst

```diff
@@ -0,0 +1,104 @@
+.. _nrf_desktop_dvfs:
+
+DVFS module
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF Desktop Dynamic Voltage and Frequency Scaling (DVFS) module is responsible for switching the frequency and voltage according to the application needs.
+DVFS is critical for balancing performance and power consumption, especially in battery-operated devices.
+
+In the nRF Desktop, the DVFS module enables the system to use the lowest possible frequency and voltage when the application performs less computation-challenging activities such as Bluetooth LE advertising.
+However, if performing more computation-consuming tasks nRF Desktop DVFS module will dynamically change to a higher frequency to increase performance.
+
+Requesting for frequency change is done by requesting clock from clock control module.
+Clock control module is responsible for choosing the right clock frequency and voltage.
+The DVFS is set by the sysctrl core and Clock control module requests are send through nrfs service DVFS.
+
+.. note::
+   In the current solution, it is assumed that no other module requests frequency changes.
+   The nRF Desktop DVFS module is the sole entity responsible for managing frequency and voltage adjustments.
+
+Supported devices
+*****************
+
+The module is only available for nrf54h20 SoC.
+Each device has a different set of available operating points.
+
+For example the nrf54h20 SoC has the following available operating points (in documentation called frequency):
+
+1. DVFS_FREQ_HIGH - high frequency - 320 MHz, 0.8 V
+#. DVFS_FREQ_MEDLOW - medium-low frequency - 128 MHz, 0.6V
+#. DVFS_FREQ_LOW - low frequency - 64 MHz, 0.5V
```

Comment:
```suggestion
* DVFS_FREQ_HIGH - high frequency - 320 MHz, 0.8 V
* DVFS_FREQ_MEDLOW - medium-low frequency - 128 MHz, 0.6V
* DVFS_FREQ_LOW - low frequency - 64 MHz, 0.5V
```

Reply:
Should we capitalize the text after dash?